src/openforms/prefill/contrib/haalcentraal/plugin.py

```diff
@@ -152,15 +153,29 @@ def get_co_sign_values(
         )
 
     def check_config(self):
+        config = HaalCentraalConfig.get_solo()
+        assert isinstance(config, HaalCentraalConfig)
+        if not config.service:
+            raise InvalidPluginConfiguration(_("Service not selected"))
+
+        client = config.service.build_client()
+
         try:
-            config = HaalCentraalConfig.get_solo()
-            assert isinstance(config, HaalCentraalConfig)
-            if not config.service:
-                raise InvalidPluginConfiguration(_("Service not selected"))
-
-            client = config.service.build_client()
-            # FIXME: haal centraal v2 client does not support GET methods
-            client.retrieve("test", "test")
+            if config.version == HaalCentraalVersion.haalcentraal13:
+                client.retrieve("ingeschrevenpersonen", "999990676")
+            else:
+                client.operation(
```

Comment:
similar remarks as above - you should use a non-existent resource/endpoint here to ensure that no real data is returned, ever.